### ISSUE TITLE
Borderless window mod still relevant, repurpose idea

### ISSUE THREAD

**Issue by Warsen** [cursey/kanan-new#165]
Thanks to the recent update, Mabinogi runs in a borderless window without the use of mods. However, the game's built-in borderless window mode assumes you want to run the game using the full screen space of your monitor. Perfectly acceptable for players who have an HD or QHD monitor, but it's not great for people who have a UHD monitor, and it's unbearable for people who have a UHD monitor and want to multi-client. Enabling the BorderlessWindow mod in Kanan after setting Borderless Window in Mabinogi caused me to crash on subsequent game loads.

I tried modifying Kanan/BorderlessWindow.cpp with a dirty fix. I changed all instances of `m_changeStyle = true;` to `m_changeStyle = false;`

Everything worked and I could go back to using the custom option. I propose we rename the mod to "Window Position", clean up the CPP, change the Style dropdown to a disable/enable checkmark, and display the Position, Size, and Monitor boxes.

Do you want to do it cursey, or should I do it and submit a pull request?

**Comment by cursey** [comment]
I use Kanan's version of borderless window because it lets me see my taskbar. I have not experienced any crashes related to how Kanan's borderless window mod works so I'm hesitant to start changing it. If the issue you're having is caused by Kanan trying to set the window style could we get by with just adding a checkbox to disable setting the style?

**Comment by Warsen** [comment]
Did you try testing the game's built-in borderless window mode at any point? Kanan-new Borderless Window mod worked fine for me on patch day until the next day I tried checking out the mode built into the game. The built-in mode just forces my game to take up 100% of the screen. I tried to get it back to the the way things were but the toothpaste was out of the tube and my game just kept crashing on startup until I set `BorderlessWindow.Choice=0` in config.txt.

**Comment by cursey** [comment]
I set the in-game borderless window mode on just to be sure and have still not experienced any crashes (restarting, changing options, changing channels, etc.)

**Comment by cursey** [comment]
Let me know if this works for you: ebecc5d8dffd48407a4c3fda0795fd98e34013dc

**Comment by Warsen** [comment]
Yes it does, but oh darn it, I'm so sorry: I believe I wasted your time with a frivolous issue. I've been sitting wanting to make this issue for over a week and I did not bother to test it again before and after to ensure that it was a real and current issue before doing my own supposed fix.

After you said that you didn't experience crashes, I went back and undid my change, recompiled and discovered that my fix was not necessary at all. It may have been an issue just on my PC on that week we got the patch but it isn't a problem now. I'm genuinely sorry to have wasted 10-30 minutes of your time. I'll refrain from doing this to anyone on github again. The patch is not needed, there was no problem.
